CoinDock-api/database/migrations/2022_07_06_082217_update_wallets_table_columns.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('wallets', function (Blueprint $table) {
+            $table->dropColumn('balance');
+            $table->renameColumn('balance_USD', 'balance')->change();
```

Comment:
It was just renamed the column name. why did you drop the column(balance)? and then rename (balance to balance_USD).

Reply:
I have followed wrong Columns and now I had Copied the original column names from main and updated it.